pieces/shared/menu-button/trigger.tsx

```diff
@@ -0,0 +1,32 @@
+import { forwardRef, KeyboardEventHandler, MouseEventHandler } from 'react';
+
+import Button from '../button';
+import moreIcon from '../more.svg';
+
+type TriggerProps = {
+  isOpen: boolean;
+  label: string;
+  onClick: MouseEventHandler<HTMLButtonElement>;
+  onKeyDown: KeyboardEventHandler<HTMLButtonElement>;
+};
+
+const Trigger = forwardRef<HTMLButtonElement, TriggerProps>(function Trigger(
+  { isOpen, label, onClick, onKeyDown },
+  ref,
+) {
+  return (
+    <Button
+      role="button"
+      aria-haspopup="menu"
+      aria-expanded={isOpen}
+      aria-label={label}
+      onClick={onClick}
+      onKeyDown={onKeyDown}
+      ref={ref}
+    >
+      <img {...moreIcon} alt="" draggable={false} />
```

Comment:
should we add a `role="presentation"` to the image?

Reply:
I don't think that is needed if we use `alt=""` (https://www.w3.org/WAI/tutorials/images/decorative/)